packages/grids/Container.jsx

```diff
@@ -1,31 +1,65 @@
+/* eslint-disable security/detect-object-injection */
 import React from "react";
+import { cnb } from "cnbuilder";
 
 /**
  * Container package
  *
  * @param {string} children The child elements that sit inside the container
  * @param {string} width The width of the container
- * @param {string} paddingX Include padding for the container
+ * @param {string} paddingX Horizontal padding for the container
+ * @param {string} paddingY Vertical padding for the container
  * @returns {JSX.Element}
  * @constructor
  */
-export function Container({ children, width = "large", paddingX = true }) {
-  const hasChildren = children !== undefined;
 
-  // CSS for these is defined in global.css to allow for override on basic story
-  const widthClasses = new Map();
-  widthClasses.set("narrow", "su-container-narrow");
-  widthClasses.set("large", "su-container-large");
-  widthClasses.set("wide", "su-container-wide");
-  widthClasses.set("full", "su-container-full");
+// CSS for these is defined in global.css to allow for override on basic story
+const widthClasses = {
+  narrow: "su-container-narrow",
+  large: "su-container-large",
+  wide: "su-container-wide",
+  full: "su-container-full",
+  /**
+   * Centered container class from Decanter
+   * https://github.com/SU-SWS/decanter/blob/main/src/plugins/components/layout/centered-container.js
+   */
+  cc: "su-cc",
+};
+
+/**
+ * Padding options based on the Decanter responsive spacing system.
+ * Base is the smallest step and 10 is the largest.
+ * https://github.com/SU-SWS/decanter/blob/main/src/plugins/components/responsive-spacing/responsive-spacing.js
+ */
+const paddingYClasses = {
+  none: "",
+  base: "su-rs-py-0",
+  1: "su-rs-py-1",
+  2: "su-rs-py-2",
+  3: "su-rs-py-3",
+  4: "su-rs-py-4",
+  5: "su-rs-py-5",
+  6: "su-rs-py-6",
+  8: "su-rs-py-8",
+  10: "su-rs-py-10",
+};
+
+export function Container({
+  children,
+  width = "large",
+  paddingX = true,
+  paddingY = "none",
+}) {
+  const hasChildren = children !== undefined;
 
   return hasChildren ? (
     <div
-      className={[
+      className={cnb(
         "su-mx-auto su-component-container",
-        widthClasses.get(width.toLowerCase()),
+        widthClasses[width],
         paddingX ? "su-container-px" : "",
-      ].join(" ")}
+        paddingYClasses[paddingY]
+      )}
```

Comment:
@iamrentman @broleomo My approach for adding Decanter responsive padding. I choose to add this as an option in the Container wrapper that's used for almost every component. By passing in paddingY="10" for example and pair it with the enum selector in the manifest as a content editor option, we'll be able to add flexible padding to components easily in the future.

Reply:
Slick! Love it.